catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/JsonModelBuilder.java

```diff
@@ -219,6 +219,14 @@ public JsonModelBuilder setValue(String value) {
     return this;
   }
 
+  @Override
+  public FlatFilterBuilder setDistance(Double distance) {
+    verifyResultNotYetRetrieved();
+    verifyTerminalNodeInProgress();
+    nodeInProgress.setDistance(distance);
+    return this;
+  }
```

Comment:
❓ Should we add a unit test covering this case in `JsonModelBuilderTest`?

Reply:
added